bundles/org.openhab.binding.magentatv/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -21,7 +21,6 @@
 		</channel-groups>
 
 
-		<representation-property>macAddress</representation-property>
```

Comment:
What is the new representation property?

Reply:
hmm, good catch, maybe accidentally deleted